src/coreclr/jit/likelyclass.cpp

```diff
@@ -268,7 +273,7 @@ static unsigned getLikelyClassesOrMethods(LikelyClassMethodRecord*
 
                     // Distribute the rounding error and just apply it to the first entry.
                     // Assume that there is no error If we have unknown handles.
-                    if (numberOfClasses == h.m_totalCount)
+                    if (!containsUnknownHandles)
```

Comment:
This is an unrelated change, it just that I used a wrong check in https://github.com/dotnet/runtime/pull/86965 so it didn't work properly